### PR TITLE
Edited README.md file

### PR DESCRIPTION
**Pull request by PetroKuida**
@DenisUA @yashka713 @Vitaliy-Maznyi Any suggestions on what should or should't be on readme are welcome.

### CODE REVIEW

**Comment by yashka713** [comment]
All right, but also we need to add Denis to list of members.

File: README.md [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
+## Features
 
-* System dependencies
+A description of the features will be added during the development process.
 
-* Configuration
+## Built With
 
-* Database creation
+* [The Ruby language](https://www.ruby-lang.org/) - version 2.3.1
```
Comment (DenisUA) [inline README.md]: We don't need this links

File: README.md [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
+* [Ruby on Rails](https://github.com/rails/rails) - version 5.1
+* [PostgreSQL](https://www.postgresql.org/)
 
-* Database initialization
+## Deployment
 
-* How to run the test suite
+## Authors
```
Comment (DenisUA) [inline README.md]: Please add links to githubs

**Review by DenisUA — CHANGES_REQUESTED** [review]
Please also add information about dev process. Thx

File: README.md (hunk 2 above)
Comment (PetroKuida) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -10,14 +10,37 @@ A description of the features will be added during the development process.
...
+* Ruby on Rails
+* PostgreSQL
 
 ## Deployment
 
 ## Authors
 
-* **Yaroslav Liakh**
-* **Vitaliy Maznyi**
-* **Petr Kujda**
```
Comment (Vitaliy-Maznyi) [inline README.md]: Please add badge for CI. Here is markdown:
`[Build Status](https://semaphoreci.com/ydionis/ruby-hotel)`

**Comment by Vitaliy-Maznyi** [comment]
:+1:

**Comment by DenisUA** [comment]
LGTM

**Review by DenisUA — APPROVED** [review]

**Pull request merged**